aws/resource_aws_vpc.go

```diff
@@ -235,6 +241,30 @@ func resourceAwsVpcRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("enable_classiclink", classiclink_enabled)
 	}
 
+	DescribeClassiclinkDnsOpts := &ec2.DescribeVpcClassicLinkDnsSupportInput{
+		VpcIds: []*string{&vpcid},
+	}
+
+	respClassiclinkDnsSupport, err := conn.DescribeVpcClassicLinkDnsSupport(DescribeClassiclinkDnsOpts)
+	if err != nil {
+		if awsErr, ok := err.(awserr.Error); ok && awsErr.Code() == "UnsupportedOperation" {
+			log.Printf("[WARN] VPC Classic Link DNS Support is not supported in this region")
+		} else {
+			return err
+		}
```

Comment:
Should we default this value to `false`? It looks like here in the error case the value will not get set at all (`d.Set` only called in the `else` clause here). I'm imagining a situation where a user has an account/region that doesn't support this, they try to enable it with `true` here, it fails, but we never set it ourselves to false and so they end up with `true` in the state file, which doesn't reflect reality. Is that possible or am I missing something?